piptools/scripts/compile.py

```diff
@@ -216,44 +216,46 @@ def has_arg(self, arg_name):
     show_default=True,
     type=click.Path(file_okay=False, writable=True),
 )
-@click.option("--pip-args", help="Arguments to pass directly to the pip command.")
+@click.option(
+    "--pip-args", "pip_args_str", help="Arguments to pass directly to the pip command."
+)
 @click.option(
     "--emit-index-url/--no-emit-index-url",
     is_flag=True,
     default=True,
     help="Add index URL to generated file",
 )
 def cli(
-    ctx,
-    verbose,
-    quiet,
-    dry_run,
-    pre,
-    rebuild,
-    find_links,
-    index_url,
-    extra_index_url,
-    cert,
-    client_cert,
-    trusted_host,
-    header,
-    index,
-    emit_trusted_host,
-    annotate,
-    upgrade,
-    upgrade_packages,
-    output_file,
-    allow_unsafe,
-    generate_hashes,
-    reuse_hashes,
-    src_files,
-    max_rounds,
-    build_isolation,
-    emit_find_links,
-    cache_dir,
-    pip_args,
-    emit_index_url,
-):
+    ctx: click.Context,
+    verbose: int,
+    quiet: int,
+    dry_run: bool,
+    pre: bool,
+    rebuild: bool,
+    find_links: Tuple[str],
+    index_url: str,
+    extra_index_url: Tuple[str],
+    cert: Optional[str],
+    client_cert: Optional[str],
+    trusted_host: Tuple[str],
+    header: bool,
+    index: bool,
+    emit_trusted_host: bool,
+    annotate: bool,
+    upgrade: bool,
+    upgrade_packages: Tuple[str],
+    output_file: Optional[LazyFile],
+    allow_unsafe: bool,
+    generate_hashes: bool,
+    reuse_hashes: bool,
+    src_files: Tuple[str],
+    max_rounds: int,
+    build_isolation: bool,
+    emit_find_links: bool,
+    cache_dir: str,
+    pip_args_str: Optional[str],
```

Comment:
Why did you rename it?

Reply:
Note, the CLI option is untouched , only changed variable name. See option decorator.

Reply:
There's a lot of changes and renaming variables on top of everything else is hard to follow. It's usually better to make separate patches that are easier to digest for reviewers.

Reply:
The intention was to make diff less and make PR easier for reviewers.

Reply:
Would you like me to revert the variable name back? I'm okay with both variants.